docs/index.html

```diff
@@ -0,0 +1,89 @@
+---
+layout: index
+---
+
+<section class="bg-dark py-3">
+    <div class="container">
+        <div class="text-center">
+            <h1>Collaborate on Code</h1>
+            <h2>Distributed Collaborative Development in your IDE</h2>
+        </div>
+        <div class="row mt-4 mb-1">
+            <div class="mx-auto">
+                <a href="{{ '/documentation/getting-started.html' | absolute_url }}"><button type="button"
+                        class="btn btn-getting-started btn-lg">Getting Started</button></a>
+            </div>
+        </div>
+        <div class="row my-5">
+            <div class="mx-auto">
+                <img class="img-fluid" src="{{ '/assets/images/animation/saros.gif' | relative_url }}" />
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="my-5">
+    <div class="container">
+        <div class="row py-5 justify-content-center">
+            <div class="col-12 col-sd-10 col-md-8 col-xl-6 mx-2 text-center">
+                <h2 class="mb-3">Several Concurrent Writers</h2>
+                <p class="lead">Share a project in a group of <span class="font-weight-bold">up to five team
+                        members</span> that can edit the shared files
+                    at the same time.</p>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="bg-dark mt-5">
+    <div class="container py-5">
+        <div class="text-center mb-1">
+            <h2 class="mb-3">Follow-Mode</h2>
+            <p class="lead">Continuously observe all navigation and editing activities of a team members.</p>
```

Comment:
I used "team member" because I tried to use a wording that directly understandable for users that don't know Saros. And I assumed that they don't know that the term "participant" is related to a Saros session.

Reply:
"team member" is ok as well. But it should definitely be singular with the current wording (see suggestion).